packages/common/src/ContractInteractor.ts

```diff
@@ -852,8 +897,8 @@ calculateTransactionMaxPossibleGas: result: ${result}
 
   // TODO: a way to make a relay hub transaction with a specified nonce without exposing the 'method' abstraction
   async getRegisterRelayMethod (baseRelayFee: IntString, pctRelayFee: number, url: string): Promise<any> {
-    const hub = this.relayHubInstance
-    return hub.contract.methods.registerRelayServer(baseRelayFee, pctRelayFee, url)
+    const registrar = this.relayRegistrar
```

Comment:
1. its not the "least needed". I've seen worse
2. its the convention all those methods use. maybe because of referencing all those this-prefixed, Instance-suffixed long member names...